src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm')
+      .filter(man => Math.ceil(man.died / 100) === century)
+
+    : people.filter(person => person.sex === 'm');
+
+  const age = men.map((man) => man.died - man.born);
+  const sum = age.reduce((acc, year) => acc + year, 0);
```

Comment:
you did this functionality 2 times for different arrays
let's create separate function that will accept people array and will return average age value